wiki/Client/Beatmap_editor/Menu/en.md

```diff
@@ -4,147 +4,139 @@
 
 ## Edit
 
 ![Edit menu](img/M_Edit.jpg "Edit menu")
 
-Edit commands. Same as default MS shortcuts.
-
-### Redo/Undo commands
-
-Do note that these commands can be disabled at [Options](/wiki/Client/Options) sidebar to prevent crash when editing marathon songs.
+### Redo/undo commands
 
 | Name | Description |
 | :-- | :-- |
-| Undo (`Ctrl` + `Z`) | Undo one step. |
-| Redo (`Ctrl` + `Y`) | Redo one step. |
+| Undo (`Ctrl` + `Z`) | Undo the last edit. |
+| Redo (`Ctrl` + `Y`) | Repeat the last edit. |
 
-### Edit commands (Select a note before use)
+### Edit commands
 
 | Name | Description |
 | :-- | :-- |
-| Cut (`Ctrl` + `X`) | Cut the note(s). |
-| Copy (`Ctrl` + `C`) | Copy the note(s) |
-| Paste (`Ctrl` + `V`) | Paste the note(s) |
-| Delete (`Delete`) | Delete the note(s) |
+| Cut (`Ctrl` + `X`) | Cut selected objects. |
+| Copy (`Ctrl` + `C`) | Copy selected objects. |
+| Paste (`Ctrl` + `V`) | Paste selected objects. |
+| Delete (`Delete`) | Delete selected objects. |
 
 ### Special commands
 
 | Name | Description |
 | :-- | :-- |
-| Select All (`Ctrl` + `A`) | Select every single notes. |
-| Clone (`Ctrl` + `D`) | Copy + Paste. |
+| Select All (`Ctrl` + `A`) | Select all hit objects. |
+| Clone (`Ctrl` + `D`) | Copy selected objects and paste them one [beat](/wiki/Music_theory/Beat) after the selection. |
 
 ### Rotation commands
 
 | Name | Description |
 | :-- | :-- |
-| Reverse selection (`Ctrl` + `G`) | **Slider:** Reverse the numbering placement. **Select All + this:** Your beatmap design goes the other way around. Example: A hit circle at 10 seconds will be moved to the last 10 seconds of the song and vice versa. |
-| Flip Horizontally (`Ctrl` + `H`) | Flip the note(s) from left to right (with respect to y-axis line of Playfield origin) and vice versa. |
-| Flip Vertically (`Ctrl` + `J`) | Flip the note(s) from up to down (with respect to x-axis line of Playfield origin) and vice versa. |
-| Rotate 90° Clockwise (`Ctrl` + `>`) | Rotate the note(s) 90° right |
-| Rotate 90° Anticlockwise (`Ctrl` + `<`) | Rotate the note(s) 90° left |
-| Rotate by... (`Ctrl` + `Shift` + `R`) | Set your rotation angle (+/- 180°) for the poor note(s). You can adjust the rotation Direction \[(Anti-)Clockwise\] and Origin \[(Playfield/Selection) Centre\]. |
-| Scale by... (`Ctrl` + `Shift` + `S`) | Set your enlargement size (0.500x - 1.200x) for the poor note(s). You can adjust the Origin \[(Playfield/Selection) Centre\] and "Apply to" (the enlargement scale to) either or both x-axis and/or y-axis. |
+| Reverse Selection (`Ctrl` + `G`) | **Slider:** change the direction to the opposite. **Arbitrary selection:** reorder hit objects in time, making the last one the first, and so on. |
+| Flip Horizontally (`Ctrl` + `H`) | Mirror the selection across the Y axis of the playfield (from left to right, or vice versa). |
+| Flip Vertically (`Ctrl` + `J`) | Mirror the selection across the X axis of the playfield (from up to down, or vice versa). |
```

Comment:
i *think* this still makes sense while being more brief. the names also include "horizontally" and "vertically" so i'm not sure how much the parts in parentheses are needed

```suggestion
| Flip Horizontally (`Ctrl` + `H`) | Mirror the selection across the Y axis of the playfield (left and right). |
| Flip Vertically (`Ctrl` + `J`) | Mirror the selection across the X axis of the playfield (up and down). |
```

Reply:
yeah, was too attached to the original wording